src/components/shared/userMenu.js

```diff
@@ -44,14 +44,14 @@ const CustomDialogTitle = styled(AppBar)({
   boxShadow: 'none'
 })
 
-const [msg, sendNotification] = useNotification()
-console.log(msg)
 
 export default function UserMenu ({ language, userData, token }) {
   const { i18n } = useTranslation()
   const [anchorEl, setAnchorEl] = React.useState(null)
   const [settings, setOpenSettings] = React.useState(false)
-
+  const [msg, sendNotification] = useNotification()
```

Comment:
i moved them out, because when including them in the function, storybook fails

Reply:
storybook component needs to be mocked with the higher level component on the render return.
`
 <SnackbarProvider maxSnack={5}><Mycomponent></Mycomponent></SnackbarProvider>`

i'm fixing that on a new commit, just let me have the time to add a connection lost chip.